src/Utils/utils.ts

```diff
@@ -544,3 +544,39 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 export const keysOf = <T extends object>(obj: T) => {
   return Object.keys(obj) as (keyof T)[];
 };
+
+//removes all symbols except _ and - (one part of startCase)
+export const cleanStringForNames = (str: string): string => {
+  return str.replace(/[^a-zA-Z0-9_-]+/g, " ").trim();
+};
+
+export const cleanStringForNotifications = (str: string): string => {
+  return str.replace(/[^a-zA-Z0-9]+/g, " ").trim();
+};
```

Comment:
if you are building a utility function specific to a context, let's not keep it in shared utils.tsx file and instead keep it in notification.js itself. Or consider renaming it to be generic.

```suggestion
export const cleanString = (str: string): string => {
  return str.replace(/[^a-zA-Z0-9]+/g, " ").trim();
};
```